app/services/workflow_creator.rb

```diff
@@ -28,6 +28,9 @@ def create_workflow_steps
       processes.map do |process|
         WorkflowStep.create!(workflow_attributes(process))
       end
+
+      # Create/update metadata if passed in
+      version.update_metadata if version.metadata
```

Comment:
Maybe a six-vs-half-dozen situation, but what would you feel about moving the responsibility for this check into the `VersionMetadata#update_metadata` method: https://github.com/sul-dlss/workflow-server-rails/pull/745/files#diff-61977d248b654178763e77a2fddc09152a20b85b1aecfc8222ef4233564593fcR15 Like, have it no-op if metadata is not present.

Reply:
yeah thought about that and have no opinion -- not sure if there is a benefit one way or another, unless it's more clear that way

Reply:
The potential benefit is not needing to add this check again if we wind up adding more spots where metadata is updated, but since we only have the one spot, maybe moot

Reply:
i'll move it